packages/connection-form/src/components/advanced-options-tabs/tls-ssl-tab/tls-client-certificate.tsx

```diff
@@ -41,8 +49,12 @@ function TLSClientCertificate({
             );
           }}
           showFileOnNewLine
-          optional
-          optionalMessage="Optional (required with X.509 auth)"
+          optional={optional ?? true}
+          optionalMessage={
+            optional && displayDatabaseConnectionUserHints
+              ? 'Optional (required with X.509 auth)'
```

Comment:
On the `TLS` page it looks like this message isn't being shown:
![Screen Shot 2022-04-01 at 12 37 02 PM](https://user-images.githubusercontent.com/1791149/161305905-3466d009-0c71-4bdf-9928-8dc492874430.png)

Maybe we need to pass the `displayDatabaseConnectionUserHints` from the tls tab https://github.com/mongodb-js/compass/blob/main/packages/connection-form/src/components/advanced-options-tabs/tls-ssl-tab/tls-ssl-tab.tsx#L177

Reply:
Ah sorry, this was actually just an editing leftover – `displayDatabaseConnectionUserHints` and `optional` should default to `true` in the TLS components. They now do :slightly_smiling_face: